### DISCUSSION TITLE
Could Not Read Credentials

### DISCUSSION THREAD

**Discussion by TrinityNova** [Kunzisoft/KeePassDX#932] [Q&A]
Help. I'm new to KeePass and not sure what happened.

I've spent a couple of days setting up KeePassDX on my Android. I have memorized my Master Key and disabled the Advanced Unlocking. Now it says "Could Not Read Credentials" and I can't view my passwords. Advice?

**Comment by J-Jamet** [comment]
The credential you enter is simply not the one in the database. The password must be identical and the key file must not be modified.
If you are sure of yourself, use a backup of the file.
https://github.com/Kunzisoft/KeePassDX/wiki/Credentials
https://github.com/Kunzisoft/KeePassDX/wiki/Backup
